web/src/styles/scss/print.scss

```diff
@@ -17,6 +17,17 @@ hr.subsection-rule {
   }
 }
 
+.viewonly-contractor-resources {
+  .ds-c-review {
+    border-bottom: 0 !important;
+    padding-top: 0 !important;
+  }
+
+  .ds-c-review:last-of-type {
+    padding-bottom: 0 !important;
+  }
+}
+
```

Comment:
Initially, I attempted to add `ds-u` utility classes to the base `<Review />` component to turn off the bottom border and y-padding. This did not work. Filed a bug with the design system: https://github.com/CMSgov/design-system/issues/817

I don't like throwing `!important` around, but it gets the job done, in this instance.